src/modules/Builder/pages/BuilderApplet/BuilderApplet.tsx

```diff
@@ -52,16 +57,21 @@ export const BuilderApplet = () => {
 
   const { handleFormChange } = useBuilderSessionStorageFormChange<AppletFormValues>(getValues);
 
+  const handleFormChangeDebounced = useMemo(
```

Comment:
Why memo here?

Reply:
without memo it triggers new instance each rerender and handleFormChangeDebounced does not work properly